test/test_block_store_service.py

```diff
@@ -107,7 +107,9 @@ def test_attach_detach_volume(self):
             net, subnet = helpers.create_test_network(
                 self.provider, instance_name)
             test_instance = helpers.get_test_instance(
-                self.provider, instance_name, subnet=subnet)
+                self.provider, instance_name, subnet=subnet,
+                zone=helpers.get_provider_test_data(self.provider,
```

Comment:
@MartinPaulo What if we remove the zone parameter from the signature and let the get_test_instance method internally default to using the zone from provider_test_data? This is already being done this way for machine type and image in create_test_instance, so it may make it simpler to use and consistent?